pkg/training/delete.go

```diff
@@ -30,13 +29,7 @@ func DeleteTrainingJob(jobName, namespace string, jobType types.TrainingJobType)
 	if jobType == types.UnknownTrainingJob {
 		return fmt.Errorf("Unsupport job type,arena only supports: [%v]", utils.GetSupportTrainingJobTypesInfo())
 	}
-	// 1. Handle legacy training job
-	err := helm.DeleteRelease(jobName)
-	if err == nil {
-		log.Infof("Delete the job %s successfully.", jobName)
-		return nil
-	}
-	log.Debugf("%s wasn't deleted by helm due to %v", jobName, err)
```

Comment:
Remove this for `helm.DeleteRelease` is out of date and will always return non-nil error.